model/src/main/java/oracle/kubernetes/weblogic/domain/v1/WlsDomain.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017, 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at
+// http://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.weblogic.domain.v1;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Represents information from a WLS domain that is needed in order to validate an operator domain.
+ */
+public interface WlsDomain {
```

Comment:
Who are the users of this interface? I see that WlsDomainConfig implements it but I can't seem to find anywhere else that references the WlsDomain interface.

Reply:
No, that's part of the plan to move the validation into the domain itself, but I didn't move it just yet

Reply:
I see. The changes related to wlsconfig look fine to me.